processing/data_collection/gazette/spiders/mg_itauna.py

```diff
@@ -0,0 +1,36 @@
+import datetime as dt
+
+from dateparser import parse
+from scrapy import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class MgItaunaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3133808"
+    name = "mg_itauna"
+    allowed_domains = ["itauna.mg.gov.br"]
+    start_urls = ["https://www.itauna.mg.gov.br/portal/diario-oficial/"]
+
+    def parse(self, response):
+        diarios = response.xpath("//div[@class='d_e_modelo_diario']")
+        for diario in diarios:
+            url = (
+                "https://www.itauna.mg.gov.br"
+                + diario.css("a")[1].extract().split('"')[1]
+            )
+            date = diario.xpath("div/span/text()").extract()[0].split()[0]
+            is_extra_edition = True if len(diario.css("i")) == 4 else False
```

Comment:
For the same reasons as given before I think that this check could render better results since you would be checking for the a button that explicitly contains the string "Extra" (maybe "Edição Extra" if you feel like it):

`is_extra_edition = diario.xpath(".//span[contains(./text(), 'Extra')]").get() is not None`